backend/code/archipelag/market/views.py

```diff
@@ -31,7 +32,7 @@ def market_create(request):
 
     body_data = loads(request.body.decode('utf-8'))
     if not is_fields_are_valid(body_data):
-        error = dict(error="Błąd wewnętrzny. Złe pola.")
```

Comment:
Form can take all errors in validating data. Could you use that?
If is too hard - ignore this.

Reply:
next time